scripts/mkcloud

```diff
@@ -110,7 +110,7 @@ iscloudver 7plus && : ${controller_node_memory:=12582912}
 : ${ironicnode_hdd_size:=20}
 
 # mysql (MariaDB actually) is the default option for Cloud8
-iscloudver 8plus && want_database_sql_engine="mysql"
+iscloudver 8plus && export want_database_sql_engine="mysql"
```

Comment:
If you want this variable on admin then you need to export it somewhere in `sshrun()`.

Reply:
@nicolasbock It seems that all exported variables that match `^want_` are automatically forwarded to crowbar system:

 - First the wariables are put into a local file on the host:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud#L333
```bash
env|grep ... -e ^want_ -e ... | sort >> $mkcconf
```
 - Then the file is being copied to the remote system:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud#L336
```bash
$scp -r $SCRIPTS_DIR $mkcconf root@$adminip:
```

 - On the remote system the file is being sourced:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L20
```bash
    source $mkcconf
```

Reply:
All `want_*` variables are exported in `sshrun`, but as @matelakat points out, the variable was not visible.